.github/quarkus-github-bot.yml

```diff
@@ -478,6 +478,7 @@ triage:
         - extensions/config-yaml/
         - core/deployment/src/main/java/io/quarkus/deployment/configuration/
         - core/runtime/src/main/java/io/quarkus/runtime/configuration/
+      notify: [radcortez]
```

Comment:
Maybe we could also add `title: "config"` and see if it doesn't catch too many things.

We removed it because we were matching the description at first and it was triggered for a lot of issues. Maybe just the title would work?

Reply:
Sure, I can do that on Monday

Reply:
PR updated